indigo/distribution.yaml

```diff
@@ -12684,6 +12684,21 @@ repositories:
       url: https://github.com/turtlebot/turtlebot_create_desktop.git
       version: indigo
     status: maintained
+  turtlebot_exploration_3d:
+    doc:
+      type: git
+      url: https://github.com/RobustFieldAutonomyLab/turtlebot_exploration_3d.git
+      version: indigo
+    release:
+      tags:
+        release: release/indigo/{package}/{version}
+      url: https://github.com/RobustFieldAutonomyLab/turtlebot_exploration_3d.git
+      version: 0.0.1
```

Comment:
Did you run `bloom-release` for Indigo? If not the release section will not work. You can add the doc and source sections without releasing it with bloom.